src/oscar/templates/oscar/partials/google_analytics_transaction.html

```diff
@@ -2,26 +2,27 @@
 {% load l10n %}
 
 {% block extratracking %}
-    ga('require', 'ecommerce', 'ecommerce.js');
-    ga('ecommerce:addTransaction', {
-      'id': '{{ order.number|escapejs }}',
-      'affiliation': '{{ shop_name|escapejs }}',
-      'revenue': '{{ order.total_incl_tax|unlocalize|escapejs }}',
-      'shipping': '{{ order.shipping_incl_tax|unlocalize|escapejs }}',
-      'tax': '{{ order.total_tax|unlocalize|escapejs }}',
-      'currency': '{{ order.currency|escapejs }}'
+    gtag('event', 'purchase', {
+      "transaction_id": '{{ order.number|escapejs }}',
+      "affiliation": '{{ shop_name|escapejs }}',
+      "value": '{{ order.total_incl_tax|unlocalize|escapejs }}',
+      "currency": '{{ order.currency|escapejs }}',
+      "tax": '{{ order.total_tax|unlocalize|escapejs }}',
+      "shipping": '{{ order.shipping_incl_tax|unlocalize|escapejs }}'
     });
 
     {% for line in order.lines.all %}
-    ga('ecommerce:addItem', {
-      'id': '{{ order.number|escapejs }}',
-      'name': '{{ line.title|escapejs }}',
-      'sku': '{{ line.partner_sku|escapejs }}',
-      'category': '{{ line.category|default:"Uncategorised"|escapejs }}',
-      'price': '{{ line.unit_price_incl_tax|unlocalize|escapejs }}',
-      'quantity': '{{ line.quantity|escapejs }}'
-    });
+      gtag('event', 'add_to_cart', {
```

Comment:
You're still doing `add_to_cart`? The item information need to be passed to the `purchase` event `items` key. There should be no add_to_cart event here at all.

Reply:
Oops, I had not seen the previous comment.